examples/box/contract.py

```diff
@@ -1,19 +1,18 @@
-from algopy import ARC4Contract, Box, OnCompleteAction, TransactionType, arc4, op
+from algopy import ARC4Contract, Box, OnCompleteAction, TransactionType, UInt64, arc4, op
 
 
 class BoxContract(ARC4Contract):
-
     def __init__(self) -> None:
-        self.oca = Box(OnCompleteAction)
-        self.txn = Box(TransactionType)
+        self.oca = Box(UInt64)
```

Comment:
@daniel-makerx we haven't had examples folder validation where we run puya compile against those files, runnign against this was failing claiming that OnCompleteAction is not an arc4 type and not supported. Hence changed to uints

Reply:
Good catch, however I think there might be an issue with Puya itself here, will investigate there. But this will do as a temporary workaround in this project